nav2_map_server/CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ if(BUILD_TESTING)
   add_subdirectory(test)
 endif()
 
-ament_export_include_directories(include)
+ament_export_include_directories("include/${PROJECT_NAME}")
```

Comment:
Intentional change here (ditto with voxel grid)? Can you explain why we export this way (other refactor packages require too?)

Reply:
Right, this was a bug in my two previous PRs.  I fixed them here because it is somewhat important to fix.

What this fixes is the CMake variable called `nav2_map_server_INCLUDE_DIRS`.  If any downstream consumers were using that variable, then it would point to the wrong subdirectory currently.  That is, the header files are installed to `/path/to/nav2_map_server/include/nav2_map_server/nav2_map_server`, but without this fix `nav2_map_server_INCLUDE_DIRS` points to `/path/to/nav2_map_server/include`.  That means that a `#include "nav2_map_server/foo.hpp"` wouldn't work currently, because it would have to be `#include "nav2_map_server/nav2_map_server/foo.hpp"`.

We didn't notice this in the previous PRs because `ament` prefers to use targets when they are available, so it hardly ever falls back to `nav2_map_server_INCLUDE_DIRS`.  But it is possible that there are downstream projects that do use it, so it may as well be correct.